### PR TITLE
VA-277: Add component library structure and docs

### PR DESCRIPTION
**Pull request by acabouet**

### CODE REVIEW

File: docroot/themes/custom/vagov/assets/js/src/_example.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
[the remark this answers is not in the document]
Comment (acabouet) [reply docroot/themes/custom/vagov/assets/js/src/_example.js]: Yup, removed in last commit.

File: docroot/themes/custom/vagov/gulpfile.js [hunk 2]
```diff
@@ -13,13 +23,48 @@ gulp.task('sass', function () {
...
+        .pipe(sourcemaps.write('./'))
+        .pipe(gulp.dest('assets/js'));
+});
+
+/**
+ * @task clearcache
+ * Clear all drupal caches
+ */
+gulp.task('clearcache', function(done) {
+    return cp.spawn('lando', ['drush'], ['cache-rebuild'], {stdio: 'inherit'})
```
[the remark this answers is not in the document]
Comment (acabouet) [reply docroot/themes/custom/vagov/gulpfile.js]: Thanks!

File: docroot/themes/custom/vagov/assets/js/src/_example.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (andyhawks) [reply docroot/themes/custom/vagov/assets/js/src/_example.js]: nice catch @ethanteague

**Comment by mbenziane** [comment]
Created an environment test-vagovcms#pr-2<br><br> [button_visit-site](http://internal-test-vagovcms-3000-2004441885.us-gov-west-1.elb.amazonaws.com)

**Pull request merged**